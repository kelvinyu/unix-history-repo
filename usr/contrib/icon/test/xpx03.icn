record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("every write(\"...\"(1,10,2)) ----> ",image(every write("..."(1,10,2))) | "none")
   write("every write(\"image\"(write)) ----> ",image(every write("image"(write))) | "none")
   write("\":\"(\"abcdef\",3,5) ----> ",image(":"("abcdef",3,5)) | "none")
   write("\"[]\"(&lcase,3) ----> ",image("[]"(&lcase,3)) | "none")
   write("image(proc(\"^\",1)) ----> ",image(image(proc("^",1))) | "none")
   write("image(proc(\"^\",2)) ----> ",image(image(proc("^",2))) | "none")
   write("proc(\"+\",2)(3,4) ----> ",image(proc("+",2)(3,4)) | "none")
   write("proc(proc)(\"write\") ----> ",image(proc(proc)("write")) | "none")
   write("proc(\"+\") ----> ",image(proc("+")) | "none")
   write("\"|\"(1,2) ----> ",image("|"(1,2)) | "none")
end
