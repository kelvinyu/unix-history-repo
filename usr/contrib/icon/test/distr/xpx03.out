1
3
5
7
9
every write("..."(1,10,2)) ----> none
function write
every write("image"(write)) ----> none
":"("abcdef",3,5) ----> "cd"
"[]"(&lcase,3) ----> "c"
image(proc("^",1)) ----> "function ^"
image(proc("^",2)) ----> "function ^"
proc("+",2)(3,4) ----> 7
proc(proc)("write") ----> function write
proc("+") ----> function +
Run-time error 106 at line 18 in xpx03.icn
procedure or integer expected
offending value: "|"
